pkg/kubelet/apis/deviceplugin/v1beta1/api.pb.go

```diff
@@ -251,26 +251,121 @@ func (m *ListAndWatchResponse) GetDevices() []*Device {
 	return nil
 }
 
+type TopologyInfo struct {
+	Node                 *NUMANode `protobuf:"bytes,1,opt,name=node,proto3" json:"node,omitempty"`
+	XXX_NoUnkeyedLiteral struct{}  `json:"-"`
+	XXX_sizecache        int32     `json:"-"`
+}
+
+func (m *TopologyInfo) Reset()      { *m = TopologyInfo{} }
+func (*TopologyInfo) ProtoMessage() {}
+func (*TopologyInfo) Descriptor() ([]byte, []int) {
+	return fileDescriptor_00212fb1f9d3bf1c, []int{4}
+}
+func (m *TopologyInfo) XXX_Unmarshal(b []byte) error {
+	return m.Unmarshal(b)
+}
+func (m *TopologyInfo) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
```

Comment:
I am curious about the XXX prefix - what does it represent ?

Reply:
This is in auto-generated code, so it's not really written by this project.  But I suspect it just means something like "generic object".

Reply:
How about replacing the XXX with 'Generic' ?

Reply:
This is auto-generated code (and from another project) so we can't really change it here.